src/hnsw.c

```diff
@@ -229,3 +229,20 @@ Datum       vector_l2sq_dist(PG_FUNCTION_ARGS)
 
     PG_RETURN_FLOAT8((double)vector_dist(a, b, usearch_metric_l2sq_k));
 }
+
+PGDLLEXPORT PG_FUNCTION_INFO_V1(index_from_external);
+Datum       index_from_external(PG_FUNCTION_ARGS)
+{
+    char *tablename_str = PG_GETARG_CSTRING(0);
+    char *index_path_str = PG_GETARG_CSTRING(1);
+
+    // TODO : Call the build function
+    bool index_created = false;
+    // bool index_created = build_index(tablename_str, index_path_str);
```

Comment:
You may need to call [`index_create`](https://github.com/postgres/postgres/blob/3f8c98d0b631882e3c513954df42891c1c9585c7/src/backend/catalog/index.c#L666-L732) from here.